core/src/main/scala/akka/persistence/dynamodb/query/scaladsl/DynamoDBReadJournal.scala

```diff
@@ -158,4 +173,20 @@ final class DynamoDBReadJournal(system: ExtendedActorSystem, config: Config, cfg
     dbSource
   }
 
+  // EventTimestampQuery
+  override def timestampOf(persistenceId: String, sequenceNr: Long): Future[Option[Instant]] = {
+    queryDao.timestampOfEvent(persistenceId, sequenceNr)
+  }
+
+  //LoadEventQuery
+  override def loadEnvelope[Event](persistenceId: String, sequenceNr: Long): Future[EventEnvelope[Event]] = {
+    queryDao
+      .loadEvent(persistenceId, sequenceNr, includePayload = true)
+      .map {
+        case Some(row) => deserializeBySliceItem(row)
```

Comment:
Also rename to `item`?

```suggestion
        case Some(item) => deserializeBySliceItem(item)
```